modules/apps/data-engine/data-engine-test/src/testFunctional/tests/upgrades/wem/StructuresUpgrade.testcase

```diff
@@ -11,13 +11,14 @@ definition {
 		SignIn.signIn();
 
 		Smoke.viewWelcomePage();
-
```

Comment:
@leticia-maciel as good practices, next time separate changes like this into more than one commit. Thanks!